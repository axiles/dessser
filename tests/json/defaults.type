{
  int: i8 default 42;
  float: float default 3.14;
  string: string default "foo";
  list: i24[[]] default [[]]; // only for empty lists for now
  vector: i8[3] default (make-vec 1 2 3);
  array: bool[] default []; // only for empty arrays for now
  bool: bool default true;
}
