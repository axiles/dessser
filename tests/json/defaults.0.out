(42 0x1.91eb851eb851fp+1 "foo" 0 () (1 2 3) 0 () T)
